client_wasm/demo/js/index.js

```diff
@@ -65,7 +67,10 @@ const getByteParams = async function (setupFile) {
 
 start();
 
-import proverConfig from "../../../fixture/client.origo_tcp_local.json";
+const mode = window.MODE || 'tee'; // Get the mode from window object
```

Comment:
nit: We are using `'tee'` as the default mode in a bunch of places - How about we put it into `const DEFAULT_MODE = "tee"`